Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -50,6 +50,7 @@ final class PledgeViewModelTests: TestCase {
   private let paymentMethodsViewHidden = TestObserver<Bool, Never>()
   private let pledgeAmountViewHidden = TestObserver<Bool, Never>()
   private let pledgeAmountSummaryViewHidden = TestObserver<Bool, Never>()
+  private let popToRootViewController = TestObserver<(), Never>()
```

Comment:
Seems we weren't testing `popViewController` before so added tests for it now as `popToRootViewController`.